reactor-core/src/jcstress/java/reactor/core/publisher/FluxPublishStressTest.java

```diff
@@ -445,4 +447,50 @@ public void arbiter(IIIIII_Result r) {
 			r.r6 = subscriber2.onErrorCalls.get();
 		}
 	}
+
+// TODO: uncomment me. Proper discard is not supported yet since we dont have stable
```

Comment:
WDYM? For now discard is weak and the commented test is going to fail

Reply:
we can uncomment the test once we have an explicit discard hook for hot producers. E.g.

```java
 source.publish(discardedValue -> {..})
 ...
 ```

Reply:
What I mean is just: do we have any way to show these changes improve anything? Is improper discarding the only flaw right now? It would seem there can be regular omissions to deliver items due to improper concurrency handling right now. The [original issue for refCount](https://github.com/reactor/reactor-core/issues/3487) had a test suggestion validating the delivery of signals is appropriate, can we use a similar test for publish?

Reply:
oh, right. I wanted to perform jmh test. Will post results in a moment